pkg/storage/pod.go

```diff
@@ -34,6 +36,8 @@ type podStorage struct {
 	// prev stores pod metric points from scrape preceding the last one.
 	// Points timestamp should proceed the corresponding points from last and have same start time (no restart between them).
 	prev map[apitypes.NamespacedName]map[string]ContainerMetricsPoint
+	//scrape period of metrics server
+	scrapeDuration time.Duration
```

Comment:
let's use consistant name for this value, it should be `metricResolution`.

Reply:
ok